grpc/src/test/java/io/stargate/grpc/service/ExecuteQueryTest.java

```diff
@@ -254,7 +254,7 @@ public void columnMetadata(
             .addExpected(
                 ColumnSpec.newBuilder()
                     .setName("c2")
-                    .setType(TypeSpec.newBuilder().setBasic(TypeSpec.Basic.TEXT)))
+                    .setType(TypeSpec.newBuilder().setBasic(TypeSpec.Basic.VARCHAR)))
```

Comment:
gRPC is a bit tricky because `query.proto` also defines both constants `TEXT = 0xA; VARCHAR = 0x0D;`. I don't feel comfortable removing or renaming here, since that would be a breaking change of the protocol.

Maybe we can deprecate `TEXT`, and justify keeping `VARCHAR` for 0x0D because that's the "internal" name used by the native protocol, and gRPC lives at that same low level.

Reply:
This starts to be quite confusing.. So other APIs would work with `text` and gRPC would know only `varchar`?

Reply:
Yeah ideally I would have changed it too, but we have to consider backward compatibility...

I'm starting to wonder if this issue is worth "fixing" at all... If both names are synonymous, then returning `varchar` by default is not a bug, just a different behavior from what Cassandra usually does.

Reply:
I guess if it is documented that is an option, but requires all new services to translate to `varchar`, and maybe confuse developers when CQL shell and other direct mechanisms expose `text`.
And to me it still seems like a bug to fix as gRPC was added relatively recently, although issue itself came with Stargate persistence. Too bad it was not caught with tests earlier, but it seems there has been some confusion between 2 options for longer time.

Reply:
So what's the deal now with grpc, are we keeping it in the proto, but removing support overall? I am fine in having it in a proto as deprecated as long as we will not end up in error once used.. Maybe additional INT test here would be nice?